setup.py

```diff
@@ -31,16 +31,16 @@
 @author: Stijn De Weirdt (Ghent University)
 """
 import vsc.install.shared_setup as shared_setup
-from vsc.install.shared_setup import ag, kw, sdw
+from vsc.install.shared_setup import kw, sdw
 
 PACKAGE = {
-    'version': '0.9.9',
+    'version': '1.0.0',
     'author': [sdw, kw],
     'maintainer': [sdw, kw],
     'install_requires': [
         'vsc-base >= 1.6.7',
```

Comment:
@wdpypere This needs a bump too, since you're importing from `vsc.utils.py2vs3`

Reply:
it says `vsc.utils` and I bumped `vsc-utils`?

Reply:
`vsc.utils.py2vs3` is in `vsc-base` :)

Reply:
after all these years that still makes no sense to me whatsoever